### PR TITLE
Remove standalone mode

### PR DESCRIPTION
**Pull request by Random-Liu**
This PR:
1) Removed the standalone cri-containerd mode
2) Updated all the integration test, e2e test and also ansible to use containerd directly.

I've cut a branch for standalone cri-containerd `standalone-cri-containerd`. https://github.com/containerd/cri-containerd/tree/standalone-cri-containerd

We may need to cherry-pick important bug fix into it to make sure we have a backup.

The kubernetes test-infra side change is https://github.com/kubernetes/test-infra/pull/7141, and these 2 PRs should be merged around the same time.

### CODE REVIEW

File: cluster/health-monitor.sh [hunk 1]
```diff
@@ -20,16 +20,15 @@ set -o pipefail
 # CRICTL is the path of crictl
 CRICTL=${CRICTL:-"crictl"}
 # INITIAL_WAIT_ATTEMPTS is the number to attempt, before start
-# performing health check. The problem is that cri-containerd
-# and containerd are started around the same time with health
-# monitor, they may not be ready yet when health-monitor is started.
+# performing health check. The problem is that containerd is
+# started around the same time with health monitor, they may
```
Comment (mikebrow) [inline cluster/health-monitor.sh]: /s/they/it/

Comment (Random-Liu) [reply cluster/health-monitor.sh]: Good catch. Will do.

Comment (Random-Liu) [reply cluster/health-monitor.sh]: Done.

**Comment by Random-Liu** [comment]
Will trigger node e2e again after https://github.com/kubernetes/test-infra/pull/7141 is merged.

**Comment by Random-Liu** [comment]
/test pull-cri-containerd-node-e2e
--

**Comment by jcvenegas** [comment]
@Random-Liu so now is not need to start cri-containerd and containerd daemons. The containerd binary will implement the cri inteface through containerd plugin model?

**Comment by Random-Liu** [comment]
@jcvenegas Yeah, that is the plan.

The actual cri-containerd server logic won't change.

The only thing changed are:
1) How cri-containerd is configured and bootstrapped. Previously it is a separate binary, now it is a plugin for containerd.
2) How cri-containerd talks with containerd. Previously it is through grpc api, we are going to get rid of that layer and let cri-containerd talk with containerd through direct function call.

**Pull request merged**